docs/source/en/model_doc/encoder-decoder.md

```diff
@@ -24,105 +22,88 @@ rendered properly in your Markdown viewer.
 <img alt="SDPA" src="https://img.shields.io/badge/SDPA-DE3412?style=flat&logo=pytorch&logoColor=white">
 </div>
 
-## Overview
-
-The [`EncoderDecoderModel`] can be used to initialize a sequence-to-sequence model with any
-pretrained autoencoding model as the encoder and any pretrained autoregressive model as the decoder.
+# Encoder Decoder Models
 
-The effectiveness of initializing sequence-to-sequence models with pretrained checkpoints for sequence generation tasks
-was shown in [Leveraging Pre-trained Checkpoints for Sequence Generation Tasks](https://huggingface.co/papers/1907.12461) by
-Sascha Rothe, Shashi Narayan, Aliaksei Severyn.
+[`EncoderDecoderModel`](https://huggingface.co/papers/1706.03762) has two main parts:
 
-After such an [`EncoderDecoderModel`] has been trained/fine-tuned, it can be saved/loaded just like
-any other models (see the examples for more information).
+Encoder: This part reads the input text and converts it into a set of numerical features capturing the meaning and context 
+of the input.
 
-An application of this architecture could be to leverage two pretrained [`BertModel`] as the encoder
-and decoder for a summarization model as was shown in: [Text Summarization with Pretrained Encoders](https://huggingface.co/papers/1908.08345) by Yang Liu and Mirella Lapata.
+Decoder: This part takes the numerical features from the encoder and generates the output text step by step. It uses the information from the encoder to produce meaningful and relevant output, such as a translation, a summary, or an answer.
 
-## Randomly initializing `EncoderDecoderModel` from model configurations.
+We can be use this model class to initialize a sequence-to-sequence model with any pretrained autoencoding model as the 
+encoder and any pretrained autoregressive model as the decoder.
 
-[`EncoderDecoderModel`] can be randomly initialized from an encoder and a decoder config. In the following example, we show how to do this using the default [`BertModel`] configuration for the encoder and the default [`BertForCausalLM`] configuration for the decoder.
+> [!TIP]
+> This model was contributed by [thomwolf](https://github.com/thomwolf). This model's TensorFlow and Flax versions
+> were contributed by [ydshieh](https://github.com/ydshieh).
+> Click on the Encoder Decoder models in the right sidebar for more examples of how to apply Encoder Decoder to different 
+> language tasks.
 
-```python
->>> from transformers import BertConfig, EncoderDecoderConfig, EncoderDecoderModel
+The example below demonstrates how to generate text with [`Pipeline`], [`AutoModel`], and from the command line.
 
->>> config_encoder = BertConfig()
->>> config_decoder = BertConfig()
+hfoptions id="usage">
+<hfoption id="Pipeline">
 
->>> config = EncoderDecoderConfig.from_encoder_decoder_configs(config_encoder, config_decoder)
->>> model = EncoderDecoderModel(config=config)
+```python
+import torch
```

Comment:
Would be better to adapt the `Pipeline`, `AutoModel` and `transformers-cli` examples for summarization since thats what the checkpoint was fine-tuned for